react-common/components/controls/Textarea.tsx

```diff
@@ -38,6 +40,8 @@ export const Textarea = (props: TextareaProps) => {
         readOnly,
         resize,
         wrap,
+        autoResize,
+        resizeRef,
```

Comment:
Maybe just `textareaRef`? People could use this for all sorts of things, not just resizing.